docs/config.md

```diff
@@ -93,6 +94,10 @@ This can be achieved in a few ways:
   - This field takes precedence over `storage.docdb.mongo-server-url` and `MONGO_SERVER_URL` env var.
   - The value should point to a directory that has a file named `MONGO_SERVER_URL` that contains the env var. Each time the file is updated, the new value will be read.
   - One common use case is to store the value of `MONGO_SERVER_URL` in a secret with the key `MONGO_SERVER_URL` and mount the secret at the path specified in this field. When the secret is updated, the new value will be fetched by Tekton Chains.
+- Setting the value of `storage.docdb.mongo-server-url-path` field
```

Comment:
I am sorry but i find the documentation not clear on how to configure a mondgo databse as dbstore, why not add examples
- configuration using storage.docdb.mongo-server-url-dir
storage.docdb.mongo-server-url-dir: /mnt/mongo-creds-secret
How to mount the secret to have the mondb file configuraiton to match the directory
- configuration using url
storage.docdb.mongo-server-url-path: /mnt/mongo-creds-secret/mongo-server-url

what needs to be done here ? a secret mounted there ?

then we can add priorities
Also why should we have two methods there? shouldnt be just  a configuration using url

Reply:
Yeah make sense on adding examples 👍🏻 
Will update the docs 

> Also why should we have two methods there? shouldnt be just a configuration using url

Because we cannot remove this field directly to maintain backward compatibility

Reply:
@PuneetPunamiya I still find this text very confusing
can we break down it like this way or similar
 You can provide MongoDB connection through different options 
- Using MONGO_SERVER_URL Environment Variable
User can set the MongoDB connection URL in the MONGO_SERVER_URL env var in the Chains deployment.
- Using storage.docdb.mongo-server-url field in the Chains configmap
You can set the connection URL using the storage.docdb.mongo-server-url field in the Chains configmap
This field overrides the MONGO_SERVER_URL env var.
- Using storage.docdb.mongo-server-url-dir Field
Another option is to set storage.docdb.mongo-server-url-dir, which points to a directory containing a file named MONGO_SERVER_URL. 
The directory path setting takes precedence over both storage.docdb.mongo-server-url and the MONGO_SERVER_URL env var.
For instance, if /mnt/mongo-creds-secret/MONGO_SERVER_URL contains the MongoDB URL, set storage.docdb.mongo-server-url-dir: /mnt/mongo-creds-secret.
- Using storage.docdb.mongo-server-url-path Field
You can use. storage.docdb.mongo-server-url-path field in Chains configmap to directly reference the file containing the MongoDB URL.
This field overrides all others (mongo-server-url-dir, mongo-server-url, and MONGO_SERVER_URL env var).

Reply:
Updated, can you please take a look, thanks!